packages/statsmodels/patches/Add-WASM-support.patch

```diff
@@ -0,0 +1,813 @@
+diff --git a/.github/workflows/emscripten.yml b/.github/workflows/emscripten.yml
+new file mode 100644
+index 000000000..e5716b090
+--- /dev/null
++++ b/.github/workflows/emscripten.yml
```

Comment:
Seems like this patch includes some non-necessary changes like GHA yaml file update. But if we can update the statmodels to 0.14.3 soon and remove this patch, I am okay to leave this as-is,

Reply:
Resolving this conversation since the patch was dropped entirely.